### DISCUSSION TITLE
Can I Bundle to One Folder with Non Python External Program Dependencies?

### DISCUSSION THREAD

**Discussion by yusuf-jkhan1** [pyinstaller/pyinstaller#7387] [PyInstaller]
If my python script uses the subprocess module to call other external programs like 'rclone', 'powershell', 'nssm' etc. If I include those programs executables inside the Bundle to One Folder root will I be able to then call subprocess like `subprocess.call('./folder_containg_binaries/rclone.exe')` ?

**Comment by bwoodsend** [comment]
`subprocess.call('./folder_containg_binaries/rclone.exe')` won't work because it assumes that your current working directory is the root of your application (see [docs](https://pyinstaller.org/en/stable/runtime-information.html#using-file) for how to reference data files properly).

For copying other programs, you need to make sure that, if the program has DLLs or data files, you include those too.
